rwengine/src/ai/TrafficDirector.cpp

```diff
@@ -72,6 +74,17 @@ std::vector<GameObject*> TrafficDirector::populateNearby(const glm::vec3& center
 
 	std::vector<GameObject*> created;
 
+	// Spawn vehicles at vehicle generators
+
+	for (auto& gen : world->state->vehicleGenerators) {
+		if (glm::distance2(center, gen.position) < radius * radius) {
```

Comment:
Please add a `/// @todo` that the car should not spawn if the player is looking at the spawn point.
It's only supposed to spawn the cars in that case if "alwaysSpawn" is true.